src/ansys/templates/python/osl_solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -47,6 +47,7 @@ python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
 ansys-optislang-core = {git = "https://github.com/ansys/pyoptislang.git", rev = "dd780ae"} # Temporarily pointing to the GitHub repo until next release is available.
 ansys-saf-glow = {version = "0.4.0", source = "solutions-private-pypi"}
+ansys-saf-pim-light-server = {version = "^0.2.0", source = "solutions-private-pypi"}
 ansys-saf-portal = {version = "0.5.dev0", source = "solutions-private-pypi"}
```

Comment:
On this [page](https://github.com/Solution-Applications/portal/releases), I do not see the release for ansys-saf-portal 0.5.dev0.
Is it normal?

Reply:
Yes. 0.5.dev0 has been release in PyPI only.